endpoints/setuid.go

```diff
@@ -148,6 +163,58 @@ func NewSetUIDEndpoint(cfg *config.Configuration, syncersByBidder map[string]use
 	})
 }
 
+// extractGDPRInfo looks for the GDPR consent string and GDPR signal in the GPP query params
+// first and the 'gdpr' and 'gdpr_consent' query params second.
+func extractGDPRInfo(query url.Values) (gdpr.RequestInfo, error) {
+	// Defualt GDPR values
+	gdprConsent := query.Get("gdpr_consent")
```

Comment:
What do you think about extracting the `"gdpr_consent"` string value to a const?